src/Data/Generic.purs

```diff
@@ -164,6 +166,28 @@ instance genericOrdering :: Generic Ordering where
     SProd "Data.Ordering.GT" [] -> Just GT
     _ -> Nothing
 
+instance genericNonEmpty :: (Generic (f a), Generic a) => Generic (NonEmpty f a) where
```

Comment:
Apparently not, I think it's because we're in the `Data.Generic` module. We would have been able to derive all the other instances otherwise (or most of them anyway).